build.gradle

```diff
@@ -63,13 +65,19 @@ buildscript {
     dependencies {
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.12.0'
         classpath 'com.dropbox.maven:pem-converter-maven-plugin:1.0'
+        classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:3.5.0'
     }
 }
 
 repositories {
     mavenCentral()
 }
 
+configurations {
```

Comment:
This swapping of the order of `configurations` and `dependencies` is a bit of an ugly change, but necessary since the `baseline` bit needs to be in the `configurations` block before `dependencies`